setup.py

```diff
@@ -40,6 +40,18 @@ def _find_msbuild_tool(tool="msbuild.exe", use_windows_sdk=False):
                 locappdir, vcpy27, r"x64\mt.exe")
             if os.path.exists(mtpath):
                 return mtpath      
+        if sys.version_info[:2] == (3,5):
```

Comment:
@tonyroberts I agree, but I did not find in registry this path. also not sure if windows sdk can be installed to non-default paths?! maybe @zooba can point how this path can be mapped in more generalized way? (r"C:\Program Files (x86)\Windows Kits\8.1\bin")